src/main/java/org/team199/robot2022/subsystems/Climber.java

```diff
@@ -86,147 +103,86 @@ public void periodic() {
 
     public void keepZeroed() {
         if(keepPosition) {
-            if(Math.abs(getLeftPosition()) > holdTolerance) {
-                left.set(Math.signum(getLeftPosition()) > 0 ? kSlowRetractSpeed : kSlowExtendSpeed);
+            if(Math.abs(leftEncoder.getPosition()) > holdTolerance) {
+                left.set(Math.signum(leftEncoder.getPosition()) > 0 ? MotorSpeed.slowRetract.value : MotorSpeed.slowExtend.value);
             } else {
                 left.set(0);
             }
-            if(Math.abs(getRightPosition()) > holdTolerance) {
-                right.set(Math.signum(getRightPosition()) > 0 ? kSlowRetractSpeed : kSlowExtendSpeed);
+            if(Math.abs(rightEncoder.getPosition()) > holdTolerance) {
+                right.set(Math.signum(rightEncoder.getPosition()) > 0 ? MotorSpeed.slowRetract.value : MotorSpeed.slowExtend.value);
             } else {
                 right.set(0);
             }
         }
     }
 
-    public void resetEncodersToExtended() {
-        leftEncoder.setPosition(extendPositionLeft);
-        rightEncoder.setPosition(extendPositionRight);
-    }
-
-    public void resetEncodersToRetracted() {
-        leftEncoder.setPosition(retractPositionLeft);
-        rightEncoder.setPosition(retractPositionRight);
-    }
-    public void resetEncodersToZero() {
-        leftEncoder.setPosition(0);
-        rightEncoder.setPosition(0);
-    }
-
-    public void extendLeft() {
-        left.set(kExtendSpeed);
-        SmartDashboard.putString("Left climber is", "Extending");
-        keepPosition = false;
-    }
-
-    public void extendRight() {
-        right.set(kExtendSpeed);
-        SmartDashboard.putString("Right climber is", "Extending");
-        keepPosition = false;
-    }
-
-    public void retractLeft() {
-        left.set(kRetractSpeed);
-        SmartDashboard.putString("Left climber is", "Retracting");
-        keepPosition = false;
+    //motor = -1 left or 1 right or 0 both
+    public void resetEncodersTo(EncoderPos posEnum,int motor){
+			// setEncoder[motor+1].accept(dEncoderPos.get(pos));
+      setEncoder[motor+1].accept(posEnum.value);
     }
 
-    public void retractRight() {
-        right.set(kRetractSpeed);
-        SmartDashboard.putString("Right climber is", "Retracting");
-        keepPosition = false;
+    public void moveMotors(MotorSpeed speedEnum, int motor){
+      setMotor[motor+1].accept(speedEnum.value);
```

Comment:
```suggestion
      setMotor[motor].accept(speedEnum.value);
```
Directly pass motors as array indexes